Lib/test/test_typing.py

```diff
@@ -5108,6 +5109,12 @@ def bar(self):
         self.assertIn('baz', dir(Foo[int]))
 
 
+class RevealTypeTests(BaseTestCase):
+    def test_reveal_type(self):
+        obj = object()
+        self.assertIs(obj, reveal_type(obj))
```

Comment:
Do we need to test `print` side-effect here?

Reply:
Good point. Do you happen to know if there's a helper for that?

Reply:
Sure! https://github.com/python/cpython/blob/main/Lib/test/support/__init__.py#L618-L625

Reply:
Thanks! Added now.